.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "[python]": {
+    "editor.defaultFormatter": "ms-python.black-formatter"
+  },
+  "python.formatting.provider": "none"
+}
```

Comment:
Adding a VSCode settings file may be useful.
It is not what the PR stated it does, so ideally this would be in a separate PR. But I can live with adding it here.

However, I don't use VSCode and therefore do not know what the settings do. One setting that may be an issue is the black formatter. We are not using it in this project because it seems a bit overkill in what it does. Are you seeing formatting issues being reported in VSCode due to it?